src/python/WMCore/Services/Rucio/Rucio.py

```diff
@@ -454,6 +454,27 @@ def closeBlockContainer(self, name, scope='cms'):
             self.logger.error("Exception closing container/block: %s. Error: %s", name, str(ex))
         return response
 
+    def moveReplicationRule(self, ruleId, rseExpression, scope='cms', **kwargs):
```

Comment:
I understand you only wanted logic-related review, but FYI, this method needs some corrections.